sale_order_line_description/tests/test_sale.py

```diff
@@ -16,7 +16,7 @@ def setUp(self):
         self.partner_model = self.env["res.partner"]
         self.product_model = self.env["product.product"]
         self.user_model = self.env["res.users"].with_context(
-            {"no_reset_password": True, "mail_create_nosubscribe": True}
+            **{"no_reset_password": True, "mail_create_nosubscribe": True}
```

Comment:
You can pass them as arguments like:
`with_context(no_reset_password=True, mail_create_nosubscribe=True)`

Reply:
@rousseldenis done